syntax = "proto3";

service Hello {
  rpc SayHello(HelloRequest) returns (HelloResponse);
}

message HelloRequest {
  string name = 1;
}

message HelloResponse {
  string name = 1;
  int32 age = 2;
}